File: Semestrul 4/MPP2/Problema8_FX/src/main/java/ro/mpp2025/problema8_fx/Repository/IRepoChild.java
package ro.mpp2025.problema8_fx.Repository;

import ro.mpp2025.problema8_fx.Domain.Child;

public interface IRepoChild extends IRepository<Child, Long> {
    Child findOneCNP(String cnp);
    Iterable<Child> findByEvent(Long id);
}

File: Semestrul 4/MPP2/Problema8_FX_Csharp/Problema8_FX_Csharp/Repository/IRepoChild.cs
namespace Problema8_FX_Csharp.Repository;

using Problema8_FX_Csharp.Domain;

public interface IRepoChild : IRepository<Child, long>
{
    public Child FindOneCNP(string cnp);
    public IEnumerable<Child> FindByEvent(long eventId);
}